src/app/components/Text/README.md

```diff
@@ -0,0 +1,13 @@
+# Text [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Text)
+
+## Description
+
+The 'Text' component is being used to parse all article story text. It takes [markdown](https://en.wikipedia.org/wiki/Markdown) as an input and converts it to the desired HTML markup. 
+
+## When to use this component
+
+This component is used throughout the page in visible text. It uses the [`Paragraph`](../Paragraph) styling and thus is not suitable for all text which may have different design requirements.
+
+## Accessibility notes
+
+This component has no features specifically targeted at accessibility however it does ensure that the correct HTML tags are used, such as `<i>`, `<b>` and `<p>` which should ensure that users using screen readers are aware of the desired inference.
```

Comment:
Interesting! I was always taught to use `<strong>` and `<em>`, which is the 'semantic' way of inferring meaning on your text. `<b>` and `<i>` are [visual cues only, and have no inference](https://stackoverflow.com/questions/3059149/when-to-use-strong-and-when-to-use-b) - could you rephrase this paragraph to make that clear? (The way it's currently worded suggests that `<b>` and `<i>` provide _more_ context to screen readers when the opposite is true).

Reply:
Chasing my discussion with the accessibility champions - I want to make sure I understand why these decisions were made. I think you're documenting it correctly but I'm not sure we chose the right implementation in the first place 😅. Will comment back here when I've heard more.

Reply:
Have heard back from the champions - makes sense. Editorial aren't using bold and italic for semantic meaning (i.e. it shouldn't be read out any differently to 'normal' text). It's purely a visual thing.

I'd update to this:

```suggestion
This component uses `<i>` and `<b>` (rather than `<em>` and `<strong>`) for text formatting. This is because editorial use italics and bold purely for visual reasons, e.g. as the opening paragraph of the article.

Editorial don't want to indicate that this text has strong stress emphasis or importance, hence the use of, say, [b](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b) over [strong](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong). Screen readers should read out all text with the same intonation regardless of how it is visually presented.
```